src/main/java/io/excaliburfrc/robot/RobotContainer.java

```diff
@@ -36,20 +38,24 @@ public RobotContainer() {
   private void configureButtonBindings() {}
 
   void manualButton() {
+    final int anglePiston = 2;
+
```

Comment:
unused; remove

Reply:
how is this no used??
`climber.climberManualCommand(armJoystick::getY, () -> armJoystick.getRawButton(**anglePiston**)).schedule();
`

Reply:
not*

Reply:
You're right, my bad.